src/Functions/array/arrayUniq.cpp

```diff
@@ -317,10 +317,46 @@ void FunctionArrayUniq::executeHashed(
     executeMethod<MethodHashed>(offsets, columns, {}, nullptr, res_values);
 }
 
-
 REGISTER_FUNCTION(ArrayUniq)
 {
-    factory.registerFunction<FunctionArrayUniq>();
+    FunctionDocumentation::Description description = R"(
+For a single argument passed, counts the number of different elements in the array.
+For multiple arguments passed, it counts the number of different **tuples** made of elements at matching positions across multiple arrays.
+
+For example `SELECT arrayUniq([1,2], [3,4], [5,6])` will form the following tuples:
+* Position 1: (1,3,5)
+* Position 2: (2,4,6)
+
+It will then count the number of unique tuples. In this case `2`.
+
+:::tip
+If you want to get a list of unique items in an array, you can use `arrayReduce('groupUniqArray', arr)`.
+:::
+)";
+    FunctionDocumentation::Syntax syntax = "arrayUniq(x[, ...yN])";
```

Comment:
`arrayUniq(arr1[, arr2, ..., arrN)`

Reply:
@rschu1ze In general, prefer `arr` over `x`?